.github/workflows/eventlist.yml

```diff
@@ -22,14 +26,14 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        go: [1.17.x]
+        go: [1.19.3]
     name: 'Build'
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
 
-      - name: Install go 1.17
+      - name: Install go 1.19

```

Comment:
Can we use the variable here? I.e. `Install go §{{ matrix.go }}`?
If not, I'd remove the version from the step name.

Reply:
[Addressed review comments](https://github.com/ARM-software/CMSIS-View/pull/14/commits/c0f043365bfd7f76b3f4cb4e2f9e6146237363b7)